src/findAssetPaths.js

```diff
@@ -17,5 +17,5 @@ export default function findAssetPaths(doc = document) {
     });
   });
 
-  return imgPaths.filter((url) => !isAbsoluteUrl(url));
+  return imgPaths.filter((url) => !isAbsoluteUrl(url) && url.length);
```

Comment:
What about something like this? `src=" "`

Reply:
good catch, will handle that too!